update.sh

```diff
@@ -3,9 +3,7 @@ set -e
 
 cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
-nodeVersion="$(awk '$1 == "FROM" { print $2; exit }' Dockerfile)"
-
-mongoExpressVersion="$(docker run --rm "$nodeVersion" npm show mongo-express version)"
+mongoExpressVersion="$(wget -qO- 'https://registry.npmjs.org/mongo-express' | jq -r '."dist-tags".latest')"
```

Comment:
I should probably also set `pipefail` to make sure this fails if the `wget` fails too.